examples/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+# SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
+
+if (NOT TARGET SixtyFPS::SixtyFPS)
+    find_package(SixtyFPS REQUIRED)
+    include(FetchContent)
+endif()
+
+add_subdirectory(printerdemo/cpp/)
+if (SIXTYFPS_FEATURE_INTERPRETER)
```

Comment:
I'll wait for the logs to check if this is really defined without being defined as a cached variable.

Reply:
The `-D` option that we pass to cmake on the command line directly creates/updates a cache entry, so it should work (and looks like it does :-)